src/utils/trust/historical.beneficial.owner.mapper.ts

```diff
@@ -1,24 +1,56 @@
-import * as Page from 'model/trust.page.model';
-import { Trust } from 'model/trust.model';
+import { v4 as uuidv4 } from 'uuid';
+import { ApplicationData } from '../../model';
+import * as Trust from '../../model/trust.model';
+import * as Page from '../../model/trust.page.model';
+import { getTrustByIdFromApp } from '../../utils/trusts';
 
 const mapTrustToPage = (
-  trustData: Trust | undefined,
-): Page.TrustDetails => {
-  if (!trustData) {
-    return {} as Page.TrustDetails;
+  appData: ApplicationData,
+  trustId: string,
+): Page.TrustHistoricalBeneficialOwner => {
+  const data = getTrustByIdFromApp(appData, trustId);
+
+  return {
+    trustId: data.trust_id,
+    trustName: data.trust_name,
+  };
+};
```

Comment:
Can this be removed after the common trust data mapper is added to the main branch?

Reply:
yes, when you implement common mapper it should be replaced with it (test also)